packages/playground-examples/copy/ko/JavaScript/Modern JavaScript/Async Await.ts

```diff
@@ -0,0 +1,109 @@
+//// { order: 1, target: "es5" }
+
+// 최신 JavaScript는 특별한 기능이 있는
+// 프로미스(Promise) 기반 API를 추가하여
+// 비동기 코드가 동기적으로 작동하는 것처럼
+// 콜백을 처리하는 방법을 추가했습니다.
+
+// 모든 언어 기능과 마찬가지로, 이러한 방식은 복잡성이 증가하는 대신 효율성이
+// 증가합니다: 함수를 async로 만드는 것은 반환값을 프로미스 형태로
+// 감싸주는 것을 의미합니다. 기존에는 문자열을 반환했지만
```

Comment:
```suggestion
// 감싸주는 것을 의미합니다. 기존에는 string을 반환했지만
```
타입은 그대로 사용하는게 나을 것 같습니다 :)